qanta/test/test_cachedwikipedia.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from qanta.wikipedia.cached_wikipedia import CachedWikipedia
+
+def get_wikidata(self, key: str):
+    web_page = "https://www.wikidata.org/w/api.php?action=wbsearchentities&search=" + key + "&language=en&format=json"
+    headers = {"Accept": "application/json"}
+    req = requests.get(web_page, headers=headers)
+
+    try:
+        if not req.json()['search']:
+            return None
+        else:
+            id = req.json()['search'][0]['id']
+            url = req.json()['search'][0]['concepturi']
+            headers = {"Accept": "application/json"}
+            req = requests.get(url, headers=headers)
+            claims = req.json()['entities'][id]['claims']
+            return claims
+
+    except ValueError:
+        return None
+
+class TestCachedWikipedia(unittest.TestCase):
```

Comment:
To test this we should make sure that:
1. First access of a page and wikidata attributes works
2. Using the same `CachedWikipedia` object it still works without doing a request to wikidata
3. Using a new `CachedWikipedia` object it loads it from disk instead of doing a round trip to wikidata's API.

For example, if you use python `@property` to implement this you could:

```python
cw = CachedWikipedia("data/external/wikipedia")
page = cw['albert_einstein'] # Make a request to Wikipedia API to load his page properties
print(page.claims) # Call Wikidata API to load claims, note we probably want more than just claims so this is an example
print(page.claims) # Load claims from memory

cw2 = CachedWikipedia("data/external/wikipedia")
page = cw['albert_einstein'] # Load page from disk
print(page.claims) # Load claims from disk
print(page.claims) # Load claims from memory
```